Automattic/jetpack
# RNA: IDC safe mode button

#### Changes proposed in this Pull Request:
* add the "stay in Safe mode" button to the IDC screen
* make the button send REST API request to put the site in safe mode

#### Jetpack product discussion
p9dueE-3kA-p2

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Open the database and run the following SQL query:
```delete from wp_options where option_name = 'jetpack_safe_mode_confirmed';```
2. Go to "Tools -> Connection Manager".
3. The "Safe Mode" screen should appear. In the bottom there should be the line saying  _"Or decide later and stay in Safe mode"_.
4. Click on the "Safe mode". Wait a few seconds while the API request is running, then the page should reload. There's no loader yet, we'll add that later (p9dueE-3pz-p2#comment-6234).
5. Run the following SQL query to confirm that the record was added, `option_value` should equal `1`:
```select * from wp_options where option_name = 'jetpack_safe_mode_confirmed';```
